index.html

```diff
@@ -4,8 +4,109 @@
     <meta charset="UTF-8" />
     <title>Fork Me! FCC: Test Suite Template</title>
     <link rel="stylesheet" href="css/style.css" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
   </head>
 
```

Comment:
Always try to make your design as close as possible with the main or required design as this looks not organized 😁 

![image](https://user-images.githubusercontent.com/43151556/155010277-1032a00f-16a1-4d98-bdee-e9f312bb61dd.png)